tracer.go

```diff
@@ -61,6 +62,7 @@ type Tracer struct {
 
 	baggageRestrictionManager baggage.RestrictionManager
 	baggageSetter             *baggageSetter
+	serviceNameHash           uint64
```

Comment:
don't think the tracer needs to be aware of this

Reply:
remove this?

Reply:
Oh missed this thanks